### PR TITLE
File organization

### PR DESCRIPTION
**Pull request by bradallred**
just testing the builds

### CODE REVIEW

**Comment by MarcelHB** [comment]
Want me to test this on Windows?

File: gemrb/GemRB.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #include <clocale> //language encoding
 
 #include "Interface.h"
-#include "System/Logger/Stdio.h"
+#include "Loggging/Loggers/Stdio.h"
```
Comment (Mingun) [inline gemrb/GemRB.cpp:26]:
```suggestion
#include "Logging/Loggers/Stdio.h"
```

File: gemrb/core/Interface.cpp [hunk 2]
```diff
@@ -75,7 +75,7 @@
 #include "GUI/WorldMapControl.h"
 #include "RNG.h"
 #include "Scriptable/Container.h"
-#include "System/FileStream.h"
+#include "Streams/FileStream.h"
 #include "System/FileFilters.h"
```
Comment (MarcelHB) [inline gemrb/core/Interface.cpp:80]: Below (line 85), 
```cpp
#include "CodepageToIconv.h"
```
does not work like this anymore

Comment (bradallred) [reply gemrb/core/Interface.cpp:80]: right, we probably need to make it so that cmake includes the platform directory as an include directory.

Comment (lynxlynxlynx) [reply gemrb/core/Interface.cpp:80]: Easily done at the main CMakeLists.txt:478

Comment (bradallred) [reply gemrb/core/Interface.cpp:80]: @MarcelHB did that do the trick?

File: gemrb/core/CMakeLists.txt [hunk 3]
```diff
@@ -131,8 +131,8 @@ ENDIF()
 IF(WIN32)
 	SET(gemrb_core_LIB_SRCS
 		${gemrb_core_LIB_SRCS}
-		CodepageToIconv.cpp
-		CodepageToIconvMap.cpp
+		platforms/windows/CodepageToIconv.cpp
+		platforms/windows/CodepageToIconvMap.cpp
```
[the remark this answers is not in the document]
Comment (MarcelHB) [reply gemrb/core/CMakeLists.txt]: I meant this as additional prefix: `../../platforms/windows/CodepageToIconvMap.cpp`

File: gemrb/core/Interface.cpp (hunk 2 above)
Comment (MarcelHB) [reply gemrb/core/Interface.cpp:80]: Looks good.

File: gemrb/core/CMakeLists.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (bradallred) [reply gemrb/core/CMakeLists.txt]: done. let me know if there is anything else

**Comment by MarcelHB** [comment]
Builds for me now.

**Pull request merged**